uberfire-security/uberfire-security-management/uberfire-security-management-wildfly/uberfire-security-management-wildfly10/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I added the logback config on purpose. By default (without any config) logback uses DEBUG level which pollutes the build output quite a lot.